Dashboard/app/js/lib/components/devices/AssignmentsEditView/Sidebar.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import FormSection from './screens/FormSection';
+import AddDevices from './screens/AddDevices';
+import DevicesSection from './DevicesSection';
+
+export default class Sidebar extends React.Component {
+  state = {
+    currentTab: 'ADD_DEVICE',
+  };
+
+  changeTab = tab => {
+    this.setState({ currentTab: tab });
+  };
+
+  render() {
+    return (
+      <React.Fragment>
```

Comment:
In this case I don't think we should use fragments since these are two sections that are related to each other but not exactly peers.